dev/Makefile

```diff
@@ -9,11 +9,11 @@ all: update remove clean
 # update modules
 update:
 	git submodule sync --recursive
-	git submodule update --init --force --recursive
+	git submodule update --init --force --recursive --remote
 # delete unnecessary docker modules
 remove:
 	rm -rf $(DOCKER_MODULES)
 # remove lines docker-compose
 clean:
-	head -n 43 '$(PATH_MODULES)/docker-compose.yml' > temp.yml
+	head -n 63 '$(PATH_MODULES)/docker-compose.yml' > temp.yml
 	mv temp.yml '$(PATH_MODULES)/docker-compose.yml'
```

Comment:
why is this necessary?

if you only stand up the services you need... you don't need to manipulate/change the .yml file... and this code won't have to be revisited if the source .yml files continue to change (which they will).

the 43 changing to 63 feels brittle...

Reply:
If this is included in the CRAN submission, it will trigger multiple R CMD check errors and warning. For example, path length cannot be more 100 characters (`storing paths of more than 100 bytes is not portable: `). This has btw. not changed since earlier submission, where we also used a trimmed down version.

Reply:
okay, just caught my eye.